src/dataflow/components/dataflow-program.tsx

```diff
@@ -155,7 +153,7 @@ export class DataflowProgram extends BaseComponent<IProps, IState> {
           readOnly={this.props.readOnly || !this.isReady()}
         />}
         <div className={toolbarEditorContainerClass}>
-          { !this.isGraphOnly() && <DataflowProgramToolbar
+          { showProgramToolbar && <DataflowProgramToolbar
```

Comment:
I see that we still show the program toolbar when displaying an completed relay program (no data storage block).  Not sure if that came up in this story, but that might be another case where we need to hide the toolbar.

Reply:
Good catch -- I'll add a note to the story on filtering relay programs since we don't have the mechanism to identify that case at the moment.